zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/SelectTagKeys.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2015-2018 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package zipkin2.storage.cassandra.v1;
+
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.ResultSetFuture;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Supplier;
+import zipkin2.Call;
+import zipkin2.storage.cassandra.internal.call.AccumulateAllResults;
+import zipkin2.storage.cassandra.internal.call.ResultSetFutureCall;
+
+import static zipkin2.storage.cassandra.v1.Tables.TABLE_TAGS;
+
+public class SelectTagKeys extends ResultSetFutureCall {
+
```

Comment:
just return static list from config? less code

Reply:
I am returning the static list in the TagStore here https://github.com/zeagord/zipkin/blob/b64e7950527d83f49598247ccf0f2385220cff4b/zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/CassandraTagStore.java#L32